Api/Data/Models/Thread.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SharpScape.Api.Models;
+// cant use Thread as classname cause its already used in namespace. So I name it ThreadModel
+public class ThreadModel
+{
+    [Key]
+    public Guid Id{get;set;}
+    [ForeignKey("User")]
+    public Guid UserId {get;set;}
+    public User User{get;set;}
+    public string Title { get; set; }
+    public string Body {get;set;}
```

Comment:
```suggestion
```
Essentially, I think the "body" of a thread is really just the `Post`s which belong to it

Reply:
Wait I thought Thread is like topic right? we create a new topic which have title and body(content). And people will read and comment(post) to that topic?

Reply:
I think it could be done either way, so it's decision time:
Do we want the Thread's main content to be
 - A `Thread.Body` field, and the posts are *replies to the `Thread`*, or
 - The first `Post` in the thread, and posts are *replies to other `Post`s?*

Reply:
I think option 1 is better for simplicity.